lib/stones-spec/language/gobstones.rb

```diff
@@ -28,6 +28,10 @@ def self.is_runtime_error?(result)
       result.include? 'Error en tiempo de ejecución'
     end
 
+    def self.is_syntax_error?(result)
+      result.include? 'Error de sintaxis' or result.include? 'Error en el programa'
```

Comment:
However, I would suggest two other changes:

* name it `syntax_error?`. In ruby, since you have the final question mark, adding `is_` as a prefix is redundant. 
 * Prefer always double pipe to `or`, since they are nearly the same, but all code here uses the first form